test/wallets_v2_test.js

```diff
@@ -773,6 +788,25 @@ describe("POST /v2/totp/disable_lost_device", function() {
       });
   });
 
+  it("returns the wallet after the grace period", function() {
```

Comment:
This test is in the wrong place, and IMO we can improve the name.  When you run this test, the output description is: "POST /v2/totp/disable_lost_device returns the wallet after the grace period", which is not what we're specifying.  IMO< A more accurate title would be:

POST /v2/wallets/show resets the totpKey and totpDisabled after the grace period has elapsed

and to go along with it, we should add: 

POST /v2/wallets/show resets the totpDisabled  but leaves totpKey in place before the grace period has elapsed